frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/charts/TimeSeriesChart.tsx

```diff
@@ -63,10 +63,14 @@ export const TimeSeriesChart: React.FC<TimeSeriesChart & ChartProps & ChartLineP
   }
 
   const xTickFormat = (d) => formatDate(d);
-  let xAxisStyle;
+  const xAxisFillStyle = {
+    tickLabels: { fill: 'var(--pf-global--Color--100)' },
+  };
+  let xAxisStyle: any = xAxisFillStyle;
   if (tickValues.length > 7 || width < 225) {
     xAxisStyle = {
       tickLabels: {
+        ...xAxisFillStyle.tickLabels,
```

Comment:
Why not keep it simple here? Couldn't we just inline `xAxisFillStyle` here instead of spreading it?

It's nowhere else used. :man_shrugging: 

```suggestion
        fill: 'var(--pf-global--Color--100)',
```

Reply:
updated it.